system/util/dummy_diag_publisher/README.md

```diff
@@ -0,0 +1,33 @@
+# dummy_diag_publisher
+
+## Purpose
+
+This package outputs a dummy diagnostic data for debugging and developing.
+
+## Inputs / Outputs
+
+### Outputs
+
+| Name               | Type                                    | Description                                                                                                                                                                                |
+| ------------------ | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `/diagnostics`     | `diagnostic_msgs/DiagnosticArray`       | Diagnostics outputs                                                                                                                                                                        |
+| `/diagnostics_agg` | `diagnostic_msgs::msg::DiagnosticArray` | Use aggregated diagnostic information based on [diagnostic_aggregator setting](https://github.com/tier4/autoware.iv/tree/main/system/autoware_error_monitor/config/diagnostic_aggregator). |
+| `/diagnostics_err` | `diagnostic_msgs::msg::DiagnosticArray` | This information is used to visualize HazardStatus.                                                                                                                                        |
```

Comment:
may be unnecessary

Reply:
Yes, /diagnostics_agg and /diagnostics_err have been removed here.
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/627/commits/f8d2cbd17e775c3e57542727749209b59a9460f6